components/AddToCartConfirmationModal.vue

```diff
@@ -0,0 +1,303 @@
+              {{ $t("Go to Cart") }}
+            </SfButton>
+          </div>
+        </div>
+      </div>
+    </transition>
+  </SfModal>
+</template>
+<script lang="ts">
+import { SfModal, SfBar, SfImage, SfPrice, SfButton, SfIcon } from "@storefront-ui/vue"
+import { computed, useAsync } from "@nuxtjs/composition-api"
+import { useUiState, cartGetters } from "@/composables"
+
+export default {
+  name: "AddToCartConfirmationModal",
+  components: {
+    SfModal,
+    SfBar,
+    SfImage,
+    SfPrice,
+    SfButton,
+    SfIcon,
+  },
+  setup() {
+    const { isAddToCartConfirmationOpen, toggleAddToCartConfirmationModal } = useUiState()
+    const { cart, load: loadCart } = useCart()
+    const nuxt = useNuxtApp()
+    const app = nuxt.nuxt2Context.app
+
+    useAsync(async () => {
+      await loadCart()
+    }, null)
+
+    const cartItems = computed(() => cartGetters.getItems(cart.value))
+    const cartOrder = computed(() => cartGetters.getTotals(cart.value))
+
+    const closeModal = () => {
+      toggleAddToCartConfirmationModal()
+    }
+
+    const goToCart = () => {
+      closeModal()
+      app.router.push({ path: "/cart" })
+    }
+
+    return {
+      isAddToCartConfirmationOpen,
+      toggleAddToCartConfirmationModal,
+      cartItems,
+      cartOrder,
+      goToCart,
+      closeModal,
+    }
+  },
+}
+</script>
+<style lang="scss" scoped>
+@import "~@storefront-ui/vue/styles";
+$cart-button-width: 11.6rem;
+
+.collected-product-list {
+  @include for-desktop {
+    padding: 5% 5% 2% 5%;
+  }
+}
+
+.kibo-collectedProduct {
+  border-width: 1px !important;
+
+  &__title-wrapper {
+    padding: var(--spacer-xs) 0;
+  }
+
+  &__info {
+    padding: var(--spacer-sm);
+    padding: var(--spacer-xs, 0 var(--spacer-xs) var(--spacer-xs) 0);
+  }
+
+  &__price > span {
+    font-size: var(--font-size--sm);
+    font-weight: var(--font-weight--normal);
+  }
+}
+
+.sf-property {
+  color: var(--c-black);
+  font-size: var(--font-size--sm);
+  padding: 0;
+
+  &__name {
+    font-weight: var(--font-weight--bold);
+  }
+}
+
+.sf-modal {
+  --modal-width: 34.188rem;
+  --modal-content-padding: 0;
+
+  .sf-icon.color-gray-secondary {
+    --icon-color: var(--_c-white-secondary);
+  }
+
+  @include for-desktop {
+    &__container {
+      top: 3.5rem;
+    }
+
+    .sf-bar {
+      &__title {
+        min-width: 25rem;
+      }
+    }
+  }
+}
+
+.title {
+  display: flex;
+
+  @include for-desktop {
+    padding: var(--spacer-sm) var(--spacer-lg);
+  }
+
+  &--heading {
+    font-weight: var(--font-weight--bold);
+    font-size: var(--h2-font-size);
+    padding-left: var(--spacer-sm);
+  }
+
+  .sf-icon {
+    display: inline-flex;
+
+    svg {
+      width: 1.5rem;
+      height: 1.5rem;
+    }
+  }
+}
+
+.detailed-cart {
+  padding: 1%;
+}
+
+.border-space {
+  margin-bottom: 1.3rem;
+  margin-top: 0.4rem;
+}
+
+.section-border {
+  border-bottom: 1px solid var(--c-light);
+}
+
+.sf-button--pure {
+  --button-height: 1.625rem;
+}
```

Comment:
use  .sf-button--small instead

Reply:
sf-button--pure button class is added by sfBar component, couldn't be change the class name.